waterlog/src/Components/TankGraph.js

```diff
@@ -1,44 +1,24 @@
 import React from 'react';
 import { Line, defaults } from 'react-chartjs-2';
+import {mapDataTime} from '../utils';
 
-const TankGraph = (props) => {
-  let labelX = props.props.dailytankgraph.dataPoints.map(a => (new Date(a.x).getHours() + ":00"));
-  let dataY = props.props.dailytankgraph.dataPoints.map(a => Math.round(a.y));
-  var start = new Date(Date.now());
-  var end = new Date(Date.now());
-  var startT2 = start.setTime("00:00:01");
-  var endT2 = end.setTime("23:59:00");
-  let startTime = Math.floor(startT2/1000);
-  let endTime =  Math.floor(endT2/1000);
+const TankGraph = (props) => { 
 
-  let forecast = [];
-  for(let i = startTime; i <= endTime; i = i + 3600){
-    forecast.push(1*i + 80);
-  }
-  
+  let labelX = props.props.placeholder.map(a => (new Date(a).getHours() + ":00"));
+  let myData = props.props.dailytankgraph.dataPoints;
+  let dataY = mapDataTime(myData); 
   let data = {
     labels: labelX,
-    datasets: [
-      {
-        label: 'forecast',
-        data: forecast,
-        fill: true,
-        borderColor: 'rgba(0,191,255,1)', 
-        backgroundColor: 'rgba(255,23,68,0.4)',
-        pointBackgroundColor: 'rgba(0,191,255,1)',
-        pointRadius: 5,
-        pointHitRadius: 5
-      },
-      
+    datasets: [ 
       {
         label: 'rands',
-        data: dataY,
+        data:  dataY,
         fill: true,
         borderColor: 'rgba(23,209,255,1)',
```

Comment:
you guys are mizng rgba and hex choose one and stick to it throughout the project

Reply:
Thanks its being fixed on this PR: https://github.com/Hunter27/WaterLog-Front-end/pull/106